services/web/server/src/simcore_service_webserver/utils_aiohttp.py

```diff
@@ -35,25 +35,26 @@ def get_routes_view(routes: RouteTableDef) -> str:
     return fh.getvalue()
 
 
-def create_url_for_function(request: web.Request) -> Callable:
-    app = request.app
+def create_url_for_function(
+    app: web.Application, url: URL, headers: dict[str, str]
```

Comment:
I am not sure that change makes it simpler... and that is a aiohttp util function